be/src/exprs/array_functions.cpp

```diff
@@ -160,6 +160,11 @@ class ArrayRemoveImpl {
             return (*null_map)[idx] != 0;
         };
 
+        auto* targets_col = &targets;
+        if constexpr (ConstTarget) {
+            targets_col = down_cast<const ConstColumn*>(&targets)->data_column().get();
+        }
+
```

Comment:
ArrayContainsImpl also has the same problems

Reply:
yeah, I resolved it